orca-core/src/main/java/com/netflix/spinnaker/orca/pipeline/persistence/jedis/RedisExecutionRepository.java

```diff
@@ -509,6 +517,11 @@ Execution retrieveOrchestrationForCorrelationId(
   }
 
   protected Execution buildExecution(@Nonnull Execution execution, @Nonnull Map<String, String> map, List<String> stageIds) {
+    Id serializationErrorId = registry
+      .createId("executions.deserialization.error")
+      .withTag("executionType", execution.getType().toString().toLowerCase())
+      .withTag("application", execution.getApplication().toLowerCase());
```

Comment:
We don't normally do the lowercasing anywhere else. I'm not sure we should deviate.

Reply:
Ok, consistency wins out over my love of all lowercase metrics.